tests/functional/partial_parsing/test_pp_vars.py

```diff
@@ -334,7 +334,7 @@ def test_profile_env_vars(self, project):
         with pytest.raises(FailedToConnectError):
             run_dbt(["run"], expect_pass=False)
 
-        log_output = Path(project.project_root, "logs", "dbt.log").read_text()
+        log_output = Path(project.logs_dir, "dbt.log").read_text()
```

Comment:
I ended up doing something similar in [my attempt to fix this](https://github.com/dbt-labs/dbt-core/pull/6908) but @gshank raises a really good point in [this comment](https://github.com/dbt-labs/dbt-core/pull/6908#discussion_r1100730065) - why is this test is now throwing an error in click-cli when it doesn't in main?